modules/transcription-service-workflowoperation/pom.xml

```diff
@@ -75,12 +80,6 @@
       <artifactId>easymock</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.opencastproject</groupId>
-      <artifactId>opencast-transcription-service-amberscript</artifactId>
-      <version>10-SNAPSHOT</version>
-      <scope>compile</scope>
-    </dependency>
```

Comment:
Since this has nothing to do with actual PR. I do see the value of fixing it, but I would prefer this file not to be part of this PR.